src/common/os_dimm_ndctl.c

```diff
@@ -240,7 +240,7 @@ os_dimm_usc(const char *path, uint64_t *usc)
 
 	os_stat_t st;
 	struct ndctl_ctx *ctx;
-
+	int ret = 0;
```

Comment:
Reverse the logic. By default set ret to -1, and set it back to 0 on success. It will simplify this code.